src/index.tsx

```diff
@@ -10,12 +10,14 @@ import { NetworkContextName } from './constants'
 import './i18n'
 import App from './pages/App'
 import store from 'state'
+
```

Comment:
Another place I'd keep without the line change to avoid future conflicts

```suggestion
```

Reply:
I think is the autoformat, ok, I'll try to bring it back :P